data-sources/azure-virtual-machines/config.yml

```diff
@@ -0,0 +1,9 @@
+id: azure-virtual-machines
+displayName: Azure Virtual Machines
+description: |
+  Monitor Azure Virtual Machines by connecting Azure to New Relic.
+install:
+  primary:
+    link:
+      url: https://one.newrelic.com/infra/azure?account=1&state=7ce24172-040d-d896-09ea-00f926561d6e
```

Comment:
Looks like we are trying to link to a nerdlet, specifically the VM install, from the URL. This should be changed to a nerdlet that requires an account id

Reply:
possible core data source candidate

Reply:
Another idea here is that, if its linking to a specific nerdlet we can expose that by making the primary a nerdlet.

Any additional nerdletState that is needed we can _maybe_ also supply what it needs. Question is, what would the nerdlet need since we probably don't own it

Reply:
After some discussion we wont create this datasource and just use the azure core data source